.prettierrc.json

```diff
@@ -0,0 +1,4 @@
+{
+  "trailingComma": "es5",
+  "singleQuote": true
```

Comment:
Not the biggest fan of `singleQuote` as a customisation, but hey. Do like trailing commas.